tests/wsgi/tests/test_util.py

```diff
@@ -72,7 +71,10 @@ def test_build_precache_map():
 def test_event():
     event = {
         "database": "enwiki",
-        "meta": {"topic": "mediawiki.revision-create"},
+        "meta": {
+            "topic": "eqiad.mediawiki.revision-create",
+            "schema": "mediawiki.revision-create"
```

Comment:
The schema field is now at `$schema` top level.  The event will have these like:
```
$schema: /mediawiki/revision/create/1.0.0
meta: {
  stream: mediawiki.revision-create
  ...
}